### DISCUSSION TITLE
Jest configuration?

### DISCUSSION THREAD

**Discussion by kinoli** [intlify/vue-i18n#868] [Q&A]
Does anyone have an example of how to set this up with Jest. I have this, but it doesn't seem to cover everything. Such as, it doesn't know what the i18n-t component is. It gives me a warning when I use it `[intlify] Not found parent scope. use the global scope.`

``` 
const i18n = createI18n({
  legacy: false,
  globalInjection: true,
  locale: 'en',
  fallbackLocale: 'en',
  messages: {},
  datetimeFormats: { en: datetimeFormats },
  numberFormats: { en: numberFormats }
});
config.global.plugins = [i18n];
```

**Comment by kazupon** [comment]
Please read the docs here:
https://vue-i18n.intlify.dev/guide/advanced/component.html#scope-resolving
